get_minecraft_files.js

```diff
@@ -6,18 +6,17 @@ function getMinecraftFiles (minecraftVersion, temporaryDir, cb) {
   const jarPath = temporaryDir + '/' + minecraftVersion + '.jar'
   const unzippedFilesDir = temporaryDir + '/' + minecraftVersion
   fs.mkdirpSync(unzippedFilesDir)
-  downloadClient(minecraftVersion, jarPath, function (err) {
+  downloadClient(minecraftVersion, jarPath, async (err) => {
     if (err) {
       cb(err)
       return
     }
-    extract(jarPath, { dir: unzippedFilesDir }, function (err) {
-      if (err) {
-        cb(err)
-        return
-      }
+    try {
+      await extract(jarPath, { dir: unzippedFilesDir })
```

Comment:
ah it's needed to put bakc extract to version 2 then

Reply:
yes they updated their api to have everything async last month, it broke the extractor.

Reply:
yeah my point was I just merged https://github.com/PrismarineJS/minecraft-jar-extractor/pull/21 which switched back to version 1.6, we need to undo this change

Reply:
oh, ok I'll do that